packages/perseus/src/widgets/input-number.jsx

```diff
@@ -187,22 +196,34 @@ class InputNumber extends React.Component<Props> {
             }
         }
 
-        const classes = {};
-        classes["perseus-input-size-" + this.props.size] = true;
```

Comment:
I like what you're doing here - looks like you're getting rid of global styles and unscoped class names in favor of using Aphrodite. Just a warning though: when I did something similar it broke Webapp tests because they were expecting specific class names. It seems like we're treating these class names as part of Perseus' external API (which is hinted at because they're in the `perseus-api.jsx` file).

I think these changes will move us in the right direction, but we'll need to be ready to work on the Selenium tests when releasing. I also wonder if that makes this a non-patch release? Since, in a way, the API changes? I'd say "no", but just a thought.

Reply:
My thinking is that this is that although these class names are technically visible outside of Perseus, they shouldn't be. I really don't want outside code being able to manipulate styling of a widget or component... and if we do want that, we support it explicitly via some better, more explicit API.  All that to say, I don't consider it part of our "documented" API.  

Thanks for the heads-up about tests depending on these class names. I checked webapp and there's no strings that match `perseus-input-size` so I _think_ were safe.